app/controllers/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+module V0
+  class MessageDraftsController < SMController
+    def create
+      params = draft_create_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    def update
+      params = draft_update_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    private
+
+    def draft_create_params
+      # Call to MHV message create fails if unknown field present, and does not accept recipient_id. This
+      # functionality will be moved into 'gem' once gem is moved to vets-api.
```

Comment:
This comment does not make sense to me. Can we get rid of it? I don't think there is any plan to ever move anything back in to a gem. Referencing gem here is confusing.